kustomize/resource_kustomization.go

```diff
@@ -182,12 +182,19 @@ func kustomizationResourceDiff(ctx context.Context, d *schema.ResourceDiff, m in
 	mapper := m.(*Config).Mapper
 	gzipLastAppliedConfig := m.(*Config).GzipLastAppliedConfig
 
+	// Diff Old/Modified
 	do, dm := d.GetChange("manifest")
+	tm := dm.(string)
+	// dm will always be empty if it's known after apply
+	if tm == "" {
+		return nil
+	}
+	to := do.(string)
 
 	kmm := newKManifest(mapper, client)
-	err := kmm.load([]byte(dm.(string)))
+	err := kmm.load([]byte(tm))
 	if err != nil {
-		return logError(err)
+		return logError(fmt.Errorf("failed to parse new manifest: %s [body=%#v]", err, tm))
```

Comment:
You can not print the manifest to the log. The resource may be a secret, and now you printed that to the CI/CD log. Please remove this part.